src/client/log/log.js

```diff
@@ -69,9 +72,11 @@ export class GameLog extends React.Component {
 
     for (let i = 0; i < turns.length; i++) {
       const turn = turns[i];
+      const currentPlayerNumber = i % numPlayers;
```

Comment:
I'm actually changing the code in the [phases](https://github.com/google/boardgame.io/issues/51) branch so that the playerID that made the move will be part of the action object. Can you wait till I get that in before we merge this?

The play order will no longer be round-robin (the user can customize it), so this code will break.

Reply:
Sure, I can wait with that. If the playerID becomes a part of the action object (and I assume is then logged with the action, too?) it will be easier to use that.

I didn't want to use the playerID to match to a certain color. That's why I used the round-robin approach. If the order of turns will be customizable I can't just use `modulo`, but with a unique playerID I could still assign colors from a list when a playerID is seen in a log for the first time.

Reply:
v0.16 is released, which contains the new `playerID` in each action.

Reply:
Ok, I'll take a look and send another PR.

What did you think about the color-presets in log.css? Should I keep it like that or style things in JS programmatically?

Reply:
Color presets in log.css look good! Maybe you can look into putting them inside SCSS variables? But even without that is fine!

Reply:
I've opened a [new PR](https://github.com/google/boardgame.io/pull/70) for this.